tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/InfixSuite.scala

```diff
@@ -397,21 +419,24 @@ class InfixSuite extends BaseDottySuite {
          |""".stripMargin,
       Some(
         """|{
-           |  println("hello")
-           |  ???
-           |  ??? match {
+           |  println("hello") ??? ??? match {
```

Comment:
this is the example mentioned just above unary operators (https://docs.scala-lang.org/scala3/reference/changed-features/operators.html#unary-operators) but the explanation there is unclear: it says `???` is a symbolic _identifier_, hence the second `???` can -- and seemingly does -- start an expression; is there another reason why it couldn't?